<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rank</class>
 <widget class="QDialog" name="rank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="easyModeRank">
       <property name="text">
        <string>简单模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hardModeRank">
       <property name="text">
        <string>困难模式</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="rankTable">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
